extra/2-piping.js

```diff
@@ -16,26 +16,26 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(a, b) {
+return a + b;
```

Comment:
please, 
use proper formatting:
```js
function add(a, b) {
    return a + b;
^^^
// add extra space here
}
``` 
format it manually or follow the instructions for installing Prettier in VS Code: 
https://github.com/YuliaYaremchuk/js-1-1/tree/feature/Yulia_Yaremchuk#1-install-prettier